garak/generators/nim.py

```diff
@@ -16,23 +16,24 @@
 
 
 class NVOpenAIChat(OpenAICompatible):
-    """Wrapper for NVIDIA-hosted NIMs. Expects NIM_API_KEY environment variable.
+    """Wrapper for NVIDIA NIM microservices hosted on build.nvidia.com and self-hosted.
 
-    Uses the [OpenAI-compatible API](https://docs.nvidia.com/ai-enterprise/nim-llm/latest/openai-api.html)
-    via direct HTTP request.
+    Connects to the v1/chat/completions endpoint.
+    You must set the NIM_API_KEY environment variable even if you connect to a self-hosted NIM.
 
     To get started with this generator:
-    #. Visit [https://build.nvidia.com/explore/reasoning](build.nvidia.com/explore/reasoning)
-    and find the LLM you'd like to use.
-    #. On the page for the LLM you want to use (e.g. [mixtral-8x7b-instruct](https://build.nvidia.com/mistralai/mixtral-8x7b-instruct)),
-    click "Get API key" key above the code snippet. You may need to create an
-    account. Copy this key.
-    #. In your console, Set the ``NIM_API_KEY`` variable to this API key. On
-    Linux, this might look like ``export NIM_API_KEY="nvapi-xXxXxXx"``.
-    #. Run garak, setting ``--model_name`` to ``nim`` and ``--model_type`` to
-    the name of the model on [build.nvidia.com](https://build.nvidia.com/)
-    - e.g. ``mistralai/mixtral-8x7b-instruct-v0.1``.
 
+    #. Visit https://build.nvidia.com/explore/reasoning and find the LLM you'd like to use.
+    #. On the page for the LLM you want to use (such as `mixtral-8x7b-instruct <https://build.nvidia.com/mistralai/mixtral-8x7b-instruct>`__),
+       click **Get API key** above the code snippet.
+
+       You might need to create an account if you don't have one yet.
+       Copy this key.
+    #. In your console, set the ``NIM_API_KEY`` variable to this API key.
+
+       On Linux, this might look like ``export NIM_API_KEY="nvapi-xXxXxXx"``.
+    #. Run garak, setting ``--model_type='nim.NVIDIAOpenAIChat'`` and ``--model_name`` to
+       the name of the model on build.nvidia.com, such as ``--model_name='mistralai/mixtral-8x7b-instruct-v0.1'``.
```

Comment:
Note the command line options expect whitespace not `=`:
```suggestion
    #. Run garak, setting ``--model_type 'nim.NVIDIAOpenAIChat'`` and ``--model_name`` to
       the name of the model on build.nvidia.com, such as ``--model_name 'mistralai/mixtral-8x7b-instruct-v0.1'``.
```